src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_params.py

```diff
@@ -89,6 +90,7 @@ def load_arguments(self, _):
                 add_vnet_block(c, target)
                 add_connection_string_argument(c, source, target)
                 add_customized_keys_argument(c)
+                add_connstr_props_argument(c)
                 add_opt_out_argument(c)
```

Comment:
adding parameters for cmds would change the cmd interface which is taken as a new feature for serviceconnector-passwordless, therefore the target stable version should be `3.2.0`, not `3.1.4`. `3.1.4` means a hotfix without any cmd interface updates.

More info concerning cli extension versioning can be found here: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#cli-extension-version-transition-table

Reply:
Thanks! Let me go fix that.